cmake/machine-files/kokkos/intel-gen.cmake

```diff
@@ -0,0 +1,9 @@
+include (${CMAKE_CURRENT_LIST_DIR}/generic.cmake)
+
+option(Kokkos_ARCH_INTEL_GEN "" ON)
+set(Kokkos_ENABLE_DEPRECATED_CODE_3 TRUE CACHE BOOL "")
```

Comment:
This variable is unused. Code that was deprecated in Kokkos 3.X has been removed by now.